docker-compose.yml

```diff
@@ -107,6 +107,8 @@ services:
       dockerfile: ${FRONT_DOCKERFILE}
     image: haitaton-ui
     container_name: haitaton-ui
+    environment:
+      REACT_APP_OIDC_CLIENT_ID: haitaton-ui-dev
```

Comment:
Would it be better to have a commented line here to state the possible values?

Reply:
That's a good idea. I added some documentation here.

Also, the default was accidentally changed to Suomi.fi -authentication. I changed it back to Helsinki AD.